core/templates/pages/learner-dashboard-page/community-lessons-tab.component.ts

```diff
@@ -28,6 +28,10 @@ import { WindowDimensionsService } from 'services/contextual/window-dimensions.s
 import { Subscription } from 'rxjs';
 import { I18nLanguageCodeService } from 'services/i18n-language-code.service';
 
+interface showMoreInSectionDict {
+  [section: string]: boolean;
+}
```

Comment:
Rather name this `ShowMoreInSectionDict`

Reply:
Done